wb_modbus/minimalmodbus.py

```diff
@@ -1463,6 +1463,33 @@ class InvalidResponseError(MasterReportedException):
 # ################ #
 
 
+"""
+WB-defined public part of internal minimalmodbus api
+"""
+def create_payload(
+    functioncode,
+    registeraddress,
+    value,
+    number_of_decimals,
+    number_of_registers,
+    number_of_bits,
+    signed,
+    byteorder,
+    payloadformat,
+):
+    return _create_payload(
+        functioncode,
+        registeraddress,
+        value,
+        number_of_decimals,
+        number_of_registers,
+        number_of_bits,
+        signed,
+        byteorder,
+        payloadformat
+    )
```

Comment:
```suggestion
def create_payload(*args, **kwargs):
    return _create_payload(*args, **kwargs)
```

И аналогично для остальных